### PR TITLE
Fix ONNX saving

### PR DESCRIPTION
**Pull request by joeyballentine**
Messed up in the last merged PR. Did not type properly using ort.InferenceSession and instead used Any just to avoid having another import, plus the session is not needed to save.

### CODE REVIEW

File: backend/src/nodes/model_save_nodes.py [hunk 1]
```diff
@@ -32,11 +33,14 @@ def __init__(self):
 
         self.side_effects = True
 
-    def run(self, onnx_model: bytes, directory: str, model_name: str) -> None:
+    def run(
+        self, onnx_model: Tuple[Any, bytes], directory: str, model_name: str
```
Comment (RunDevelopment) [inline backend/src/nodes/model_save_nodes.py:37]: How is this correct? The `OnnxLoadModelNode` nodes returns `bytes`, so why is this suddenly `Tuple[Any, bytes]`?

Comment (joeyballentine) [reply backend/src/nodes/model_save_nodes.py:37]: It does not, anymore 

https://github.com/joeyballentine/chaiNNer/blob/00cde1256fa1d07f0c3cfa012a8a7bbec95c5ee5/backend/src/nodes/onnx_nodes.py#L106

You must be looking at outdated code

Comment (RunDevelopment) [reply backend/src/nodes/model_save_nodes.py:37]: Indeed, I was.

**Comment by RunDevelopment** [comment]
Just an idea. Instead of changing everything to tuples and then changing everything again the next time you change anything with ONNX, how about making a class for ONNX models?

**Comment by joeyballentine** [comment]
@theflyingzamboni wants to make model wrapper classes for all the model stuff, so I'll let him do that

**Pull request merged**